src/AWSMetadataUtilities.jl

```diff
@@ -184,6 +184,44 @@ function _documentation_cleaning(documentation::String)
     return documentation
 end
 
+"""
+    _clean_uri(uri::String)
+
+Replace URI parameters with the appropriate syntax for Julia interpolation.
+
+Find all URI parameters, and apply the following replacements:
+* { => \$(
+* } => )
+* - => _
+* + => nothing
```

Comment:
```suggestion
* + => ""
```
This is more clear to me - otherwise it seems like you're replacing it with `nothing`. Also maybe throw in a test for this condition?

Reply:
```suggestion
* + => empty string
```
Or maybe this?